packages/kbn-shared-ux-components/src/page_template/solution_nav/solution_nav.tsx

```diff
@@ -138,10 +178,14 @@ export const KibanaPageTemplateSolutionNav: FunctionComponent<
               onClose={() => setIsSideNavOpenOnMobile(false)}
               side="left"
               size={FLYOUT_SIZE}
-              closeButtonPosition="outside"
+              closeButtonPosition={isCustomSideNav ? 'inside' : 'outside'}
```

Comment:
I'm not sure this is necessarily going to be true. I would allow it to be customizable by the consuming application. Though we really should be consistent for all use cases of the side-nav so users don't have to re-learn per solution.